charts/stable/common/Chart.yaml

```diff
@@ -3,15 +3,17 @@ name: common
 description: Function library for k8s-at-home charts
 type: library
 version: 3.0.0
-kubeVersion: ">=1.16.0-0"
+kubeVersion: ">=1.16"
```

Comment:
This was intentional due to https://github.com/k8s-at-home/charts/issues/590

Reply:
Ah. What an odd bug.